com.github.bilelmoussaoui.Authenticator.json

```diff
@@ -0,0 +1,122 @@
+ {
+   "app-id": "com.github.bilelmoussaoui.Authenticator",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.26",
+   "sdk": "org.gnome.Sdk",
+   "command": "authenticator",
+   "finish-args": [
+     /* X11 + XShm */
+     "--share=ipc", "--socket=x11",
+     /* Wayland */
+     "--socket=wayland",
+     /* Filesystem */
+     "--filesystem=~/.config/Authenticator:ro",
```

Comment:
The application should respect XDG and this isn't needed.

Reply:
Alright, currently there's no possible way to import a database/export it. As the tokens are stored using libsecret and I don't plan to change that anytime soon.

So in this case, I should just remove the filesystem finish-args?